pandas/tests/frame/test_analytics.py

```diff
@@ -1839,9 +1841,10 @@ def test_clip_against_series(self, inplace):
         (0, [[2., 2., 3.], [4., 5., 6.], [7., 7., 7.]]),
         (1, [[2., 3., 4.], [4., 5., 6.], [5., 6., 7.]])
     ])
-    def test_clip_against_list_like(self, inplace, lower, axis, res):
+    def test_clip_against_list_like(self, simple_frame,
+                                    inplace, lower, axis, res):
         # GH #15390
-        original = self.simple.copy(deep=True)
+        original = simple_frame.copy(deep=True)
```

Comment:
note for later to clean up the names of these fixtures to be meaningful (e.g. float_frame is, simple_frame not so much, prob need to wait to see how much more things ike this are used)